src/Microsoft.Fx.Portability/ObjectModel/IPackageFinder.cs

```diff
@@ -20,6 +20,17 @@ public interface IPackageFinder
         /// If 'true' is returned, but no packages in the list, it means the package is not supported on the given framework.
         /// If 'false' is returned, it means we don't have any info about that assembly.
         /// </returns>
-        bool TryFindPackage(string assemblyInfo, IEnumerable<FrameworkName> targets, out ImmutableDictionary<FrameworkName, IEnumerable<NuGetPackageId>> packages);
+        bool TryFindPackage(string assemblyInfo, IEnumerable<FrameworkName> targets, out ImmutableList<NuGetPackageInfo> packages);
```

Comment:
This returns a collection; `TryFindPackages` would be clearer.

Reply:
Fixed